docs/core/get-started.md

```diff
@@ -1,51 +1,83 @@
 ---
 title: Get started with .NET Core
 description: Find resources to learn how to build .NET Core applications on Windows, Linux and macOS.
-author: johalex
-ms.author: johalex
-ms.date: 09/14/2017
+author: thraka
+ms.author: adegeo
+ms.date: 06/27/2018
 ---
+
 # Get started with .NET Core
 
-.NET Core runs on [Windows](#windows), [Linux](#linux), and [macOS / OS X](#os-x--macos).
+This article provides information on getting started with .NET Core. .NET Core can be installed on Windows, Linux, and macOS. You can code in your favorite text editor while your code produces cross-platform libraries and applications. 
+
+If you're unsure what .NET Core is, or how it relates to other .NET technologies, start with the [.NET Core Guide](./). Put simply, .NET Core is an open-source, cross-platform, implementation of .NET.
+
+First, download and install the [.NET Core SDK](https://www.microsoft.com/net/download/) on your computer.
+
+Next, open a terminal such as **PowerShell**, **Command Prompt**, or **bash**.
+
+Type the following `dotnet` commands to create, build, and run, an application.
 
-## Windows
+```console
+dotnet new console --language c# --output sample1
```

Comment:
Sounds good. However, if I remove the `--language` option, I don't think I should call out "C# application" in the next feedback comment. If the default changes at some point, then the doc would have to change too.

Reply:
If that happens, we'd have to make this change everywhere. All tutorials that use dotnet new assume C# is the default. So if we had to change, it would be fine.